play/play.go

```diff
@@ -159,15 +163,12 @@ func (m model) Update(msg tea.Msg) (r tea.Model, cmd tea.Cmd) {
 		m.w = msg
 	}
 
-	m.learn, cmd = m.learn.Update(msg)
-	m.b.Learn.SetEnabled(!m.learn.Showed())
```

Comment:
This `SetEnabled` logic must remain. It's nessesarry for "open learn info"/"close learn info" toggling

```go
	m.b.Learn.SetEnabled(!m.learn.Showed())
	m.b.CloseLearn.SetEnabled(m.learn.Showed())
```